spec/__snapshots__/PreviewEntry.test.js.snap

```diff
@@ -0,0 +1,64 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Preview Entry Components should render 1`] = `
+.c0 {
+  margin: 25px 50px 25px 50px;
+}
+
+.c1 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.c2 {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  -webkit-flex-direction: column;
+  -ms-flex-direction: column;
+  flex-direction: column;
+}
+
+.c3 {
+  -webkit-flex-direction: row;
+  -ms-flex-direction: row;
+  flex-direction: row;
+}
+
+.c4 {
+  color: #B3B3B3;
+}
+
+<div
+  className="c0"
+>
+  <div
+    className="c1"
+  >
+    <img
+      className="c2"
+      src="https://dteamdp.s3-us-west-2.amazonaws.com/pug11.jpg"
+    />
+    <div
+      className="c3"
+    >
+      Maureen
+      <br />
+      <span
+        className="c4"
+      >
+        September 2014
+      </span>
+    </div>
+  </div>
+  111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
```

Comment:
This is weird, I'm assuming it's correct since this is an autogenerated file. But it's weird.

Reply:
@ertdfgcb the text on the screen changes depending on the length of the review and I want to test for that so I made the code generate a string of 180+ '1's XD